components/FriendPages.js

```diff
@@ -28,6 +28,9 @@ const StyledName = styled.p`
 `;
 
```

Comment:
On a general remark: the name `FirendsPages` is quite confusing as this is not a components in your pages folder. Maybe FriendsList is a better name? Same for `FriendPagePreview` (FriendPreview?)